test/Serialization/private_import.swift

```diff
@@ -24,22 +24,26 @@
 // NEGATIVE-NOT: UnknownCode
 
 #if BASE
-
-  private struct Base {
-  }
-
 #elseif  CLIENT
 
-  @_private(sourceFile: "private_import.swift") import private_import
+  @_private(sourceFile: "private_import_other.swift") import private_import
 
   extension Base {
     private func foo() {}
   }
+
+  extension Base {
+    // This should not cause a failure.
+    private func shouldNotBeVisible() {}
```

Comment:
That wouldn't cause a failure anyway; shadowing is always okay. You have to have a conflict within the same module.